src/Libraries/RevitNodes/Filter/OverrideGraphicSettings.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using Autodesk.Revit.DB;
+using DynamoServices;
+using Autodesk.DesignScript.Runtime;
+using Revit.GeometryConversion;
+using RevitServices.Persistence;
+using RevitServices.Transactions;
+using System.Collections.Generic;
+
+namespace Revit.Filter
+{
+    /// <summary>
+    /// Override Graphic Settings
+    /// </summary>
+    public class OverrideGraphicSettings
+    {
+
+        #region Internal Properties
+
+        /// <summary>
+        /// Internal reference to the Revit Element
+        /// </summary>
+        internal Autodesk.Revit.DB.OverrideGraphicSettings InternalOverrideGraphicSettings
+        { 
+            get; set;
+        }
+
+        /// <summary>
+        /// Reference to the Element
+        /// </summary>
+        internal OverrideGraphicSettings(Autodesk.Revit.DB.OverrideGraphicSettings internalOverrideGraphicSettings)
+        {
+            this.InternalOverrideGraphicSettings = internalOverrideGraphicSettings;
+        }
+
+        #endregion
+
+        #region Public static constructors
+
+        /// <summary>
+        /// Create a OverrideGraphicSettings element
+        /// </summary>
+        /// <param name="cutFillColor">Fill color</param>
+        /// <param name="projectionFillColor">Projection color</param>
+        /// <param name="cutLineColor">Cut line color</param>
+        /// <param name="projectionLineColor">Projection line color</param>
+        /// <param name="cutLineWeight">Cut line weight</param>
+        /// <param name="projectionLineWeight">Projection line weight</param>
+        /// <param name="cutFillPattern">Cut fill pattern</param>
+        /// <param name="projectionFillPattern">Projection fill pattern</param>
+        /// <param name="cutLinePattern">Cut line pattern</param>
+        /// <param name="projectionLinePattern">Projection line pattern</param>
+        /// <returns>OverrideGraphicSettings</returns>
+        public static OverrideGraphicSettings ByProperties(
+            DSCore.Color cutFillColor = null,
```

Comment:
do these nulls work correctly for default values, last I remember was that this caused a partial application of the node instead of running automatically. Maybe this behavior was fixed?